website/docs/r/page_rule.html.markdown

```diff
@@ -59,7 +59,7 @@ Action blocks support the following:
 * `host_header_override` - (Optional) The Host Header to override on the origin servers.
 * `resolve_override` - (Optional) Override the origin server with this host.
 * `rocket_loader` - (Optional) Whether to set the rocket loader to `"off"`, `"manual"`, or `"automatic"`.
-* `security_level` - (Optional) Whether to set the security level to `"essentially_off"`, `"low"`, `"medium"`, `"high"`, or `"under_attack"`.
+* `security_level` - (Optional) Whether to set the security level to `"off"`, `"essentially_off"`, `"low"`, `"medium"`, `"high"`, or `"under_attack"`.
```

Comment:
"off" is an enterprise only feature - do we need to make note of this somewhere? Or just rely on the API to return sane error messages if it isn't available?

Reply:
I don't _think_ it's the responsibility of the provider to know or care about a customer's contract type with Cloudflare.  The API should just To The Right Thing ™️  if a user without appropriate permissions tries to use it.